.github/workflows/daily.yml

```diff
@@ -790,12 +790,15 @@ jobs:
         ./runtest-cluster ${{github.event.inputs.cluster_test_args}}
 
   test-macos-latest:
-    runs-on: macos-latest
```

Comment:
i don't like the pinning here for a specific version (which i assume is newer than macos-latest).
i suggest instead of keep using macos-latest here in daily.yaml.

and add a quick (maybe just compilation CI step, like we do in CI.yml) on both newer (13) and older (10?) MACOS.

@sundb WDYT?

Reply:
but what if we some day do some change that is incompatible with 11 and won't notice?

Reply:
do you mean 10.6 instead of 11?

Reply:
i mean that same as we discovered now, a compilation issue with a certain SDK or OS version, it could be some some future change we'll make will be incompatible with an older version (either pre 10.6, or 11.0), and then i'd like the CI to detect that before (or shortly after) merging.

Reply:
daily.yml: https://github.com/sundb/redis/blob/5a3b7aaaf9b4f04705c26c1affc545fe768d5ee6/.github/workflows/daily.yml#L33-L60
run CI: https://github.com/sundb/redis/actions/runs/6314536381/job/17145038951

@oranagra That's what you're supposed to want.
But I still have my doubts, there's no way to predict when macos will come up with broken compatibility changes, and we could still end up with what we're experiencing now.

Reply:
what you did seems right.
of course we cant protect from their compatibility changes introduced by future versions.
but at least i want to protect from our changes breaking compatibility with an older version.
please make a PR